### ISSUE TITLE
API for change browser's header/address bar color

### ISSUE THREAD

**Issue by DaniellMesquita** [HelloZeroNet/ZeroNet#1667]
> Styromaniac · styromaniac@zeroid.bit
> https://stackoverflow.com/questions/26960703/how-to-change-the-color-of-header-bar-and-address-bar-in-newest-chrome-version-o
> 
> Styromaniac · styromaniac@zeroid.bit 
> @Nofish: Can you make it so theme-color works in default view?

**Comment by DaniellMesquita** [comment]
@styromaniac

**Comment by Thunder33345** [comment]
should be easy, as of right now most sites just have purple accent on vivaldi(due to deriving it off icon if that's not set)

**Comment by styromaniac** [comment]
Question: Should we use site.webmanifest, manifest.json, or content.json for this?

**Comment by DaniellMesquita** [comment]
@HelloZeroNet @shortcutme

**Comment by DaniellMesquita** [comment]
> Question: Should we use site.webmanifest, manifest.json, or content.json for this?

No, not good to use content.json for things that aren't related to ZeroNet

**Comment by Thunder33345** [comment]
probably must be in the content.json since the actual site is iframed
i dont think the iframed accent will affect the top frame???
not sure i dont really work with these that much to know and be able to tell

**Comment by styromaniac** [comment]
I think it will work similarly to setting the background and viewport properties.
